apps/antalmanac/src/components/RightPane/SectionTable/SectionTable.tsx

```diff
@@ -152,49 +168,29 @@ const SectionTable = (props: SectionTableProps) => {
+            </Box>
 
             <TableContainer component={Paper} style={{ margin: '8px 0px 8px 0px' }} elevation={0} variant="outlined">
-                <Table className={classes?.table} size="small">
+                <Table size="small" style={{ minWidth: `${tableMinWidth}px` }}>
                     <TableHead>
                         <TableRow>
-                            <TableCell classes={{ sizeSmall: classes?.cellPadding }} className={classes?.row} />
-                            {Object.entries(tableHeaderColumns)
-                                .filter(([column]) => activeColumns.includes(column as SectionTableColumn))
-                                .map(([column, label]) => {
-                                    return (
-                                        <TableCell
-                                            classes={{ sizeSmall: classes?.cellPadding }}
-                                            className={classes?.row}
-                                            key={column}
-                                        >
-                                            {label !== 'Enrollment' ? (
-                                                label
-                                            ) : (
-                                                <div className={classes?.flex}>
-                                                    <span className={classes?.iconMargin}>{label}</span>
-                                                    {!isMobileScreen && (
-                                                        <Tooltip
-                                                            title={
-                                                                <Typography>
-                                                                    Enrolled/Capacity
-                                                                    <br />
-                                                                    Waitlist
-                                                                    <br />
-                                                                    New-Only Reserved
-                                                                </Typography>
-                                                            }
-                                                        >
-                                                            <Help fontSize="small" />
-                                                        </Tooltip>
-                                                    )}
-                                                </div>
-                                            )}
-                                        </TableCell>
-                                    );
-                                })}
+                            <TableCell padding="none" />
+
+                            {tableHeaderColumnEntries
+                                .filter(([column]) => getActiveColumns().includes(column as SectionTableColumn))
```

Comment:
Same idea here. I don't really like calling "getter" functions explicitly because they should ideally be calculated and captured in state. The state variable should properly react to changes (which is a given when using Zustand) and the component will update accordingly. A raw function call can't do that, and it's just run on every render. Which means that it can miss updates or over-render depending on the component's other state variables.

```suggestion
                                .filter(([column]) => selectedColumns.includes(column as SectionTableColumn))
```